features.json

```diff
@@ -46,6 +46,7 @@
     "SNAPCHAT_CUSTOM_AUDIENCE": true,
     "MARKETO_STATIC_LIST": true,
     "CAMPAIGN_MANAGER": true,
-    "SENDGRID": true
+    "SENDGRID": true,
+    "PAGERDUTY": true
```

Comment:
why we are doing the router transformation for this destination? Are we using intermediate axios call?

Reply:
updated